nilearn/datasets/func.py

```diff
@@ -1995,7 +1995,8 @@ def _fetch_development_fmri_functional(participants, data_dir, url, verbose):
 
 
 def fetch_development_fmri(n_subjects=None, reduce_confounds=True,
-                           data_dir=None, resume=True, verbose=0):
+                           data_dir=None, resume=True, verbose=0,
+                           adults_or_children = 'both'):
```

Comment:
```suggestion
                           adults_or_children='both'):
```

PEP8 style guides says that for keyword arguments there should not be spaces on the side of "="